cockatrice/src/carddatabase.h

```diff
@@ -398,11 +397,11 @@ class CardDatabase : public QObject
 
     LoadStatus loadStatus;
 
+    /*
+     * The available card database parsers
```

Comment:
👎 IMO we should avoid "what is this code" style comments unless it's very hard. In this case naming the variable `availableParsers` would be enough.